Bottom: f5ece016eb7abecde7b364d7344a1cfa17dd7776
Top:    95e9651a367c6e9887338d7ef1f2482a4b670f78
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-06 16:04:19 -0500

Refresh of add-raw-repositories

---
diff --git a/ansible-playbook/group_vars/all/nexus3.yml b/ansible-playbook/group_vars/all/nexus3.yml
index 064c2bbb..ee94374b 100644
--- a/ansible-playbook/group_vars/all/nexus3.yml
+++ b/ansible-playbook/group_vars/all/nexus3.yml
@@ -119,14 +119,32 @@ nexus_repos_raw_hosted:
   - name: ntel-sstate
     strict_content_validation: false
     write_policy: allow  # one of "allow", "allow_once" or "deny"
+  - name: orphans
+    strict_content_validation: false
+    write_policy: allow  # one of "allow", "allow_once" or "deny"
   - name: ptxdist-ipkg
     strict_content_validation: false
     write_policy: allow  # one of "allow", "allow_once" or "deny"
 
 nexus_repos_raw_proxy:
+  - name: debian
+    remote_url: 'http://ftp.us.debian.org/debian/'
+    strict_content_validation: false
+  - name: gnu
+    remote_url: 'http://mirror.team-cymru.com/gnu/'
+    strict_content_validation: false
+  - name: kernel
+    remote_url: 'https://mirrors.edge.kernel.org/pub/'
+    strict_content_validation: false
+  - name: sourceforge
+    remote_url: 'http://downloads.sourceforge.net/project'
+    strict_content_validation: false
   - name: ubuntu-archive
     remote_url: 'http://archive.ubuntu.com/ubuntu/'
     strict_content_validation: false
+  - name: xorg
+    remote_url: 'http://xorg.mirrors.pair.com/'
+    strict_content_validation: false
 
 nexus_repos_raw_group: []
